src/test/kotlin/cacophony/utils/FileInputTest.kt

```diff
@@ -0,0 +1,311 @@
+package cacophony.utils
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import java.io.File
+
+class FileInputTest {
+    private fun createFileWithContent(content: String): String {
+        val file = File.createTempFile("test_file", ".cac")
```

Comment:
I think you don't have to specify filenames, as it will get some random number appended. But it's okay.